skills/dff_voice_skill/scenario/condition.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+from df_engine.core import Context, Actor
+import common.dff.integration.context as int_ctx
+
+logger = logging.getLogger(__name__)
+
+
+def voice_message_detected(ctx: Context, actor: Actor, *args, **kwargs) -> bool:
+    voice = int_ctx.get_last_human_utterance(ctx, actor).get("annotations", {}).get("voice_service", {})
```

Comment:
but you can have its anntations but they will be default ones 
[{"sound_type": atype, "sound_duration": duration, "sound_path": path, "captions": CAP_ERR_MSG}]
So you ALSO need to check caption if it is NOT a default one

Reply:
checks added